managed-apps-vision/README.md

```diff
@@ -0,0 +1,30 @@
+
+With this change, more teams and more people are working on apps, overall. This means we can spend more time on each app, and provide a better user experience overall - we can start working more _deeply_.
+
+As some examples of what working more deeply on apps means:
+- Working upstream to fix specific customer issues, or provide feedback to maintainers on issues with upgrade paths.
+- Building more application specific monitoring and alerting, so Giant Swarm is aware of application level issues before customers.
+- Improving support for automatic scaling, so apps don't OOM on larger clusters, or scale on custom metrics.
+- Building dashboards (or reusing upstream dashboards) and making them available to customers.
+- Working with the incident and postmortem workflows to ensure that issues with applications are learnt from and improved upon.
+
+These are just examples - it is up to each team to identify how to provide a better experience with the apps they manage. Overall, we can should be aiming to improve our operations of managed apps, and reduce the amount of work our users need to do.
+
+There are going to be situations in the future where we need to work directly with customers on managed apps, as the line of responsibilities becomes better defined. We should identify where we can help reduce the operational work customers need to do, and then do so - managed apps means we help with the management of apps.
+
+Upgrades are one example. Giant Swarm can work with customers to help discover what is blocking customers from upgrading, and then help remove these blockers, such as with major version upgrades that require user interaction. Ideally, all customers run on the latest version of apps. We can improve our automation in future to start rolling out minor and patch versions automatically.
+
+Giant Swarm should ensure that new versions of applications are available as quickly as possible - ideally, within hours of new versions being available upstream,
+
+As another example, one customer cannot use automatic scaling for ingress controllers, as their network traffic spikes with certain events too quickly for automatic scaling to deal with. This means they need to manually scale beforehand. We can provide a better experience - for example, providing a system that allows for scheduling a scaling event in future. This would reduce the user's overall operational load, and provide a more managed experience.
+
+These are examples of a more general idea - identifying customer problems with managed apps, working together to find a solution, and then building that feature. Ideally, this could even be pushed or integrated upstream, or provided as a generic, open-source solution.
```

Comment:
I would add that the step from solution to feature is not a MUST step but one that we take when we see that a certain functionality makes sense for more than just a one-off and we want to drive it towards a real product (feature)

Reply:
Does that new / last sentence look good to you?